splunk_connect_for_snmp/snmp/manager.py

```diff
@@ -72,8 +73,13 @@
     "TCP-MIB",
     "UDP-MIB",
 ]
+
 logger = get_task_logger(__name__)
 
+if PYSNMP_DEBUG:
+    from pysnmp import debug
+    debug.setLogger(debug.Debug('all', options={'loggerName': logger}))
```

Comment:
I'm debating whether to add also parameter to set which logs we want, instead of always having 'all'. What do you think?